src/docs/output-targets/custom-elements.md

```diff
@@ -37,24 +37,34 @@ import { HelloWorld } from 'my-library/dist/components/hello-world';
 customElements.define('hello-world', HelloWorld);
 ```
 
-The output directory will also contain an `index.js` file which exports all of
-your components and their respective `defineCustomElement` helper functions,
-looking something like this:
+The output directory will also contain an `index.js` file which exports some helper methods by default. The contents of the file
+will look something like (the contents may look different if [`customElementsExportBehavior`](#customelementsexportbehavior) is specified):
```

Comment:
Consider moving the content in the parenthesis to a separate sentence after this section. ATM it reads a little funky:

```
The contents of the file will look something like:

<CODE_EXAMPLE/>

Note, the contents may look different if [`customElementsExportBehavior`](#customelementsexportbehavior) is specified.